types/tektelic/edoctorV1/default.schema.json

```diff
@@ -0,0 +1,62 @@
+{
+  "$id": "https://akenza.io/tektelic/edoctorV1/default.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Default",
+  "type": "object",
+  "topic": "default",
+  "processingType": "uplink_decoder",
+  "properties": {
+    "skinTemperature": {
```

Comment:
We could technically use one of our default data-models:
https://github.com/akenza-io/device-type-library/blob/main/data-models/climate/schema.json#L8